crates/biome_js_analyze/src/semantic_analyzers/nursery/use_sorted_classes.rs

```diff
@@ -0,0 +1,180 @@
+    ///
+    /// #### functions
+    ///
+    /// If specified, strings in the indicated functions will be sorted. This is useful when working with libraries like [`clsx`](https://github.com/lukeed/clsx) or [`cva`](https://cva.style/).
+    ///
+    /// Tagged template literals are also supported, for example:
+    ///
+    /// ```js
+    /// tw`px-2`;
+    /// tw.div`px-2`;
+    /// ```
+    ///
+    /// NOTE: tagged template literal support has not been implemented yet.
+    ///
+    /// ### Sort-related
+    ///
+    /// NOTE: at the moment, this rule does not support customizing the sort options. Instead, the default Tailwind CSS configuration is hard-coded.
+    ///
+    /// ## Differences with [Prettier](https://github.com/tailwindlabs/prettier-plugin-tailwindcss)
+    ///
+    /// The main key difference is that Tailwind CSS and its Prettier plugin read the `tailwind.config.js` file, which Biome can't access. Instead, Biome implements a simpler version of the configuration. The trade-offs are explained below.
+    ///
+    /// ### Values are not known
+    ///
+    /// The rule has no knowledge of values such as colors, font sizes, or spacing values, which are normally defined in a configuration file like `tailwind.config.js`. Instead, the rule matches utilities that support values in a simpler way: if they start with a known utility prefix, such as `px-` or `text-`, they're considered valid.
+    ///
+    /// This can result in false positives, i.e. classes that are wrongly recognized as utilities even though their values are incorrect. For example, if there's a `px-` utility defined in the configuration, it will match all of the following classes: `px-2`, `px-1337`, `px-[not-actually-valid]`, `px-literally-anything`.
+    ///
+    /// ### Custom additions must be specified
+    ///
+    /// The built-in Tailwind CSS preset (enabled by default) contains the set of utilities and variants that are available with the default configuration. More utilities and variants can be added through Tailwind CSS plugins. In Biome, these need to be manually specified in the Biome configuration file in order to "extend" the preset.
+    ///
+    /// ### Presets can't be modified
+    ///
+    /// In Tailwind CSS, core plugins (which provide the default utilities and variants) can be disabled. In Biome, however, there is no way to disable parts of a preset: it's all or nothing. A work-around is to, instead of using a preset, manually specify all utilities and variants in the Biome configuration file.
+    ///
+    /// ### Whitespace is collapsed
+    ///
+    /// The Tailwind CSS Prettier plugin preserves all original whitespace. This rule, however, collapses all whitespace (including newlines) into single spaces.
+    ///
+    pub(crate) UseSortedClasses {
+        version: "next",
+        name: "useSortedClasses",
```

Comment:
I just had a thought about this earlier. I don't like using Tailwind because, although it'll be the primary use case for the foreseeable future, the rule supports any utility-first CSS framework out there, including UnoCSS and even any custom one that a user might have (which is more common than it seems). But the ambiguity problem remains.

I would suggest something like `useSortedUtilityClasses` instead since it's more descriptive. And anyway, ultimately this will need to be moved to the formatter (imho! but I think @ematipico agrees) so I don't think it's worth bikeshedding too much. Let me know if you like my suggestion!

Reply:
I would like to give a name that is very precise, because once we land CSS support, this name could mislead users. Maybe `useSortedAttributeUtilityCssClasses`

- attribute -> JSX/HTML attributes
- utility -> It targets a specific pattern of styling components
- CSS
- classes -> it uses classes 

Maybe we could remove `CSS` as an alternative, if that's too verbose: `useSortedAttributeUtilityClasses`

Reply:
Attribute is unfortunately too narrow as css classes are also sorted in a bunch of other places including function calls and template strings. Also might be confusing because people could be thinking of attributes and JSX props as different things.

Agree with everything else though!

Reply:
Another point is that classes should also be sorted in CSS (`@apply`) once there's support for it.

Reply:
Thank you for the additional context! We can decide the rule name later, it doesn't have to be now :)